test/test_typing.py

```diff
@@ -0,0 +1,150 @@
+# based on NumPy numpy/typing/tests/test_typing.py
+
+import itertools
+import os
+import re
+import shutil
+from typing import IO, Dict, List
+
+import pytest
+import numpy as np
```

Comment:
Please do not assume, that numpy is always available, always check https://github.com/pytorch/pytorch/blob/b8e6e2971cba779daf14652d05f427544ab635b5/torch/testing/_internal/common_utils.py#L360
before importing

Reply:
I think I can remove the numpy dependency

Reply:
Done!